src/components/message-item/message-item.tsx

```diff
@@ -57,6 +57,7 @@ const MessageItem: React.FC<IMessageItemProps> = React.memo(
     const linkedMessageUserCreator = useSelector(
       getUserSelector(message?.linkedMessage?.userCreatorId),
     );
+    const isMessageStateInQueued = message.state === MessageState.QUEUED;
```

Comment:
@yanVojak please change variable name to isMessageQueued to make it more clear

Reply:
It will have done